src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.2.cpp

```diff
@@ -100,6 +150,26 @@ void MultiDiracDeterminant::BuildDotProductsAndCalculateRatios(int ref,
 #endif
 }
 
+void MultiDiracDeterminant::mw_BuildDotProductsAndCalculateRatios(int nw,
+                                                                  int ref,
+                                                                  RefVector<ValueVector>& ratios_list,
```

Comment:
Change this to std::vector then det0_list is not needed at all.

Reply:
I am not sure I understand how to do this. Ratio_list is already a ValueVector. I cannot use std::vector on it...